.gitignore

```diff
@@ -0,0 +1,95 @@
+*$py.class
+*.code-workspace
+*.cover
+*.egg
+*.egg-info/
+*.log
+*.manifest
+*.mo
+*.pot
+*.py,cover
+*.py[cod]
+*.retry
+*.sage.py
+*.so
+*.spec
+.AppleDB
+.AppleDesktop
+.AppleDouble
+.DS_Store
+.DocumentRevisions-V100
+.LSOverride
+.Python
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+._*
+.apdisk
+.cache
+.com.apple.timemachine.donotpresent
+.coverage
+.coverage.*
+.dmypy.json
+.eggs/
+.env
+.fseventsd
+.hypothesis/
+.installed.cfg
+.ipynb_checkpoints
+.mypy_cache/
+.nox/
+.pyre/
+.pytest_cache/
+.pytype/
+.ropeproject
+.scrapy
+.spyderproject
+.spyproject
+.tox/
+.venv
+.vscode/
+.webassets-cache
+/site
+ENV/
+Icon
+Icon
+MANIFEST
+Network Trash Folder
+Temporary Items
+__pycache__/
+__pypackages__/
+build/
+celerybeat-schedule
+celerybeat.pid
+cover/
+coverage.xml
+db.sqlite3
+db.sqlite3-journal
+develop-eggs/
+dist/
+dmypy.json
+docs/_build/
+downloads/
+eggs/
+env.bak/
+env/
+htmlcov/
+instance/
+ipython_config.py
+lib/
+lib64/
+local_settings.py
+nosetests.xml
+parts/
+pip-delete-this-directory.txt
+pip-log.txt
+pip-wheel-metadata/
+profile_default/
+sdist/
+share/python-wheels/
+target/
+var/
+venv.bak/
+venv/
+wheels/
```

Comment:
This is a very impressive list . . .

Reply:
I'm using a VSCode extension that lets you add/append gitignores from here: https://github.com/github/gitignore, specifically ones for Python, Ansible, and MacOS.
It can be verbose, so I did a `sort|uniq` and removed comments.